# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ash/destdir.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

  - Rene Rebe <rene@exactcode.de>

--- ash-linux-0.2/ash-linux-0.2/Makefile	2005-02-02 11:09:38.579275000 +0100
+++ ash-linux-0.2/Makefile	2005-02-02 11:13:54.344393016 +0100
@@ -27,8 +27,8 @@
 	$(CC) -o $(PROG) $(OBJS) $(LDFLAGS)
 
 install: all
-	install sh /bin/ash
-	install -m 644 sh.1 /usr/man/man1/ash.1
+	install sh $(DESTDIR)/bin/ash
+	install -m 644 sh.1 $(DESTDIR)/usr/man/man1/ash.1
 
 parser.o: token.def
 token.def: mktokens
